README.md

```diff
@@ -131,44 +141,61 @@ $ source ~/.bashrc
 $ virtualenv -p `which python3` ~/.venvs/userdocs
 ```
 
-Every time you want to work on this repository,
-you need to activate that virtualenv in your terminal:
+With the virtual environment created, activate it in the current
+terminal:
 
 ```
-$ source ~/.venvs/userdocs/bin/activate
+$ . ~/.venvs/userdocs/bin/activate
 ```
 
-Now that the virtual environment is created, we can
-install the dependencies into it:
+and install the dependencies into it:
 
 ```
 $ pip install -r requirements.txt
-$ ( cd rug_pygments && python setup.py install )
 ```
 
-You can now serve the documentation locally by running:
+Finally, install [HTMLProofer][html-proofer].
 
 ```
-$ mkdocs serve
+$ bundle install
```

Comment:
This is the gem bundler I guess? It's not on my system, and we didn't talk about installing it.